### ISSUE TITLE
2021.2 support

### ISSUE THREAD

**Issue by illetschko** [Vincent-P/gruvbox-intellij-theme#53]
Message from WebStorm:
Plugin 'Gruvbox Theme' (version '0.5.1') is not compatible with the current version of the IDE, because it requires build 211.* or older but the current build is WS-212.4746.80

**Comment by brnhrdt** [comment]
Also not running with IDEA 2021.2... My eyes are hurting now without this great color scheme! :)

**Comment by Huelse** [comment]
same in IDEA 2021.2

**Comment by Vincent-P** [comment]
I'll check what's wrong later today.
It is weird because in `graddle.properties` untilBuild is set to 211.* ...

**Comment by Vincent-P** [comment]
I just published a new update to support 2021.2 👍 

EDIT: You can download it manually here https://plugins.jetbrains.com/plugin/12310-gruvbox-theme/versions/stable/129891 while the update is being reviewed by Jetbrains. It usually takes 1 or 2 days.

**Comment by Vincent-P** [comment]
I'm going to reopen this issue so that people don't start new ones until the 0.5.2 update is approved by Jetbrains.

**Comment by illetschko** [comment]
UPDATE at the end! (works now....)

Well, 0.5.2 is now approved by JetBrains.
And yes, it now supports the 2021.2 versions.
Right now I updated my WebStorm and IntelliJ_Ultimate.

BUT: on both IDEs I had to RE-INSTALL the Gruvbox Theme.

If I look into "Settings"/"Plugins" before updating (see attached screen shots) I see GruvboxTheme version 0.5.2 in "Marketplace" and 0.5.1 in "Installed". But no update option. Even menu "Help"/"Check for Plugin Updates..." says: "No plugin updates available". Same after IDE update to 2021.2.

"uninstall"/"install" of Gruvbox Theme works.

Is this maybe because you had previously `pluginVersion = 1.0.0` in `gradle.properties`? If yes, since `pluginVersion = 0.5.2` looks more correct, I can live with reinstalling.
 
I work with an Ubuntu 20.04.2 LTS with Gnome

![plugins_marketplace](https://user-images.githubusercontent.com/62110395/127649726-56e700db-a653-4327-962d-190238937056.png)
![plugins_installed](https://user-images.githubusercontent.com/62110395/127649727-acdf709a-aaaa-4d6e-803f-be03f7450284.png)

UPDATE: After I noticed that it works on a test installation with the upcoming Debian 11, I re-checked it with DataGrip (which I have still on 2021.1.3). On first start, I have the same picture under "Settings"/"Plugins": 0.5.2 in "Marketplace" and 0.5.1 in "Installed". But menu "Check for Plugin Updates..." now reports "... update availaible". But only after restarting DataGrip it is also in "Settings"/"Plugins":
![plugins_with_update](https://user-images.githubusercontent.com/62110395/127992817-c5a20a3c-b4ef-4dcc-a6a4-0345f2995a9a.png)


Looks like I am simply too impatient.....
